src/app/user-settings/user-settings.component.html

```diff
@@ -0,0 +1,102 @@
+<div class="form-group m-0 px-lg-5 py-lg-2">
+  <div class="input-group w-50">
+    <label>Select User:&nbsp;</label>
+    <select title="Users" class="form-control col-xs-1" [(ngModel)]="selUser">
+      <option *ngFor="let user of users" [ngValue]="user">{{user.username}}</option>
+    </select>
+  </div>
+</div>
+
+<div class="form-group m-0 px-lg-5 d-block">
```

Comment:
I would like to avoid doing exactly the same as hte current frontend.
specially on user management. The original one is really not easy to manage and I think we could do something way better !